mozjs/src/lib.rs

```diff
@@ -10,6 +10,8 @@
     non_snake_case,
     improper_ctypes
 )]
+#![cfg_attr(feature = "crown", feature(register_tool))]
+#![cfg_attr(feature = "crown", register_tool(crown))]
```

Comment:
I am wondering if we should put this behind cfg rather than having a feature, but using a feature makes it easier to use in servo.